backend/src/Designer/Models/FooterFile.cs

```diff
@@ -1,5 +1,6 @@
 using System.Collections.Generic;
 using System.Text.Json.Serialization;
+using JetBrains.Annotations;
 
 public class FooterFile
```

Comment:
Whoops, didn't catch this when I looked at this file before 🙈 This file is missing a namespace, should probably add that.

Reply:
Wops, nice catch. fixed now 😬